packages/swarm-components/src/LinkButton.jsx

```diff
@@ -96,11 +96,11 @@ const LinkButton = (props: LinkButtonProps): React.Element<'a'> => {
 					{right ? (
 						<>
 							{children}
-							<Icon shape={icon} size="xs" color={FILLS[buttonType]} />
+							<Icon shape={icon} size="xs" />
```

Comment:
@hsbacot You forgot to remove the `FILLS` import from line 5 of this file. This will produce an error everywhere LinkButton is used.